pom.xml

```diff
@@ -331,11 +340,13 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <!-- Temporarily host coveralls SNAPSHOT with clover support locally  -->
             <plugin>
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
-                <version>4.3.0</version>
+                <version>4.4.0-SNAPSHOT</version>
```

Comment:
Given this is a snapshot version, will this prevent cutting a release (since snapshots are not allowed for releases?)

Reply:
Yep. It shouldn't be merged until (a) we either host the snapshot version somewhere (in the event the coveralls maven plugin folks drag their feet or (b) they release the 4.4.0 version of the plugin.